account_fiscal_year/models/date_range.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api, _, exceptions
+
+
+class DateRange(models.Model):
+    _inherit = 'date.range'
+
+    @api.multi
+    def unlink(self):
+        """
+        Cannot delete a date_range of type 'fiscal_year'
+        """
```

Comment:
Yes. I think both can be useful. One for not being able to delete `date.range.type` and another one for not being able to delete a `date.range` that use a `date.range.type` of type `fiscal_year`.
I will add the one on `date.range.type`.
Should I keep the other one on `date.range` ?